app/views/dashboard/_errata_widget.html.erb

```diff
@@ -1,5 +1,5 @@
 <h4 class="header">
-  <%= _("Latest Errata") %>
+  <%= _("Errata") %>
```

Comment:
Without a column in the widget that indicates the errata update date, I think you lose contextual information here. These aren't just errata, they are specifically errata sorted by updated date aka latest.